src/Workspaces/Core/Portable/FindSymbols/ReferenceLocation.cs

```diff
@@ -43,7 +42,7 @@ public struct ReferenceLocation : IComparable<ReferenceLocation>, IEquatable<Ref
         internal bool IsWrittenTo => ValueUsageInfo.IsWrittenTo();
 
         /// <summary>
-        /// Value usage info for this reference.
+        /// Symbol usage info for this reference.
         /// </summary>
         internal ValueUsageInfo ValueUsageInfo { get; }
```

Comment:
yup.  this comment indicates how both the enum name and the property name now seem wrong.  Either we should update both, or we shoudl add a new enum and reference that here, with a comment saying taht SymbolUsageInfo can be used when ValueUsageInfo is .None.

Reply:
I had actually done the rename, and found this caused lots of files to change, so reverted the rename, but forgot to sync the comments. I will clean up all of this so we use symbol usage info terminology everywhere.